src/libraries/System.Collections.Immutable/tests/ImmutableSetTest.cs

```diff
@@ -40,7 +40,23 @@ public void AddRemoveLoadTest()
         [Fact]
         public void RemoveNonExistingTest()
         {
-            this.RemoveNonExistingTest(this.Empty<int>());
+            var emptySet = this.Empty<int>();
```

Comment:
Nit: var => whatever type this is.  Same goes for the other occurrences of var throughout your changes where the right-hand side isn't a ctor or an explicit cast.

Reply:
Ah, you just moved this code.  Ok, don't worry about fixing vars in existing code.